static-checks/rpmbuild-ctest/main.fmf

```diff
@@ -6,10 +6,16 @@ environment+:
 duration: 1h
 require+:
```

Comment:
You are also missing `python3-pip`.

Reply:
Actually, in RHEL, `python3` depends on `python3-pip`, so we're good, .... but non-default python versions don't, ie. `python3.12` installs by default without `python3.12-pip`, so maybe I would still add it to keep track of that dependency explicitly.

Your choice.